scipy/stats/tests/test_stats.py

```diff
@@ -4452,14 +4452,24 @@ def test_normalitytests():
     # numbers verified with R: dagoTest in package fBasics
     st_normal, st_skew, st_kurt = (3.92371918, 1.98078826, -0.01403734)
     pv_normal, pv_skew, pv_kurt = (0.14059673, 0.04761502, 0.98880019)
+    pv_skew_less, pv_kurt_less = 1 - pv_skew / 2, pv_kurt / 2
+    pv_skew_greater, pv_kurt_greater = pv_skew / 2, 1 - pv_kurt / 2
     x = np.array((-2, -1, 0, 1, 2, 3)*4)**2
     attributes = ('statistic', 'pvalue')
 
     assert_array_almost_equal(stats.normaltest(x), (st_normal, pv_normal))
     check_named_results(stats.normaltest(x), attributes)
     assert_array_almost_equal(stats.skewtest(x), (st_skew, pv_skew))
+    assert_array_almost_equal(stats.skewtest(x, alternative='less'),
+                              (st_skew, pv_skew_less))
+    assert_array_almost_equal(stats.skewtest(x, alternative='greater'),
+                              (st_skew, pv_skew_greater))
     check_named_results(stats.skewtest(x), attributes)
     assert_array_almost_equal(stats.kurtosistest(x), (st_kurt, pv_kurt))
+    assert_array_almost_equal(stats.kurtosistest(x, alternative='less'),
+                              (st_kurt, pv_kurt_less))
+    assert_array_almost_equal(stats.kurtosistest(x, alternative='greater'),
+                              (st_kurt, pv_kurt_greater))
```

Comment:
The tests seem right. 

I suppose if i think about it, I can see that the skewness of `x = np.array((-2, -1, 0, 1, 2, 3)*4)**2` should be positive. It is not obvious to me that I would expect the kurtosis to be less than that of the normal distribution. In any case, I wish the original tests were more obvious.

For example:
```python3
a = skewnorm.rvs(a=1, size = 10000)  # skew parameter is 1 > 0
print(skewtest(a, alternative='greater'))
```
should have a small p-value

and

```python3
a = laplace.rvs(size = 1000)  # excess kurtosis of laplace is 3 > 0
print(kurtosistest(a, alternative='greater'))
```
should have a small p-value.

But I checked these locally and they looks good, so I leave it up to you whether you want to add them. I suppose this is more of a suggestion for the original authors than this PR.

Reply:
You are right. I will add the tests that are easier to understand. Thanks for the suggestion.

Reply:
The tests you have suggested here have been added now.